### DISCUSSION TITLE
Favourite books / tab management.

### DISCUSSION THREAD

**Discussion by jmzeph** [sumatrapdfreader/sumatrapdf#3600] [Ideas]
I have many books, I like to automatically open simultaneously when I use Sumatra.
These books are stored in many different storage locations. 

1. to have list of favourite books. - Favourite books.
2. to list these favourite books into a group. - a group of favourites.
3. groups to be organised by named groups. e.g. "java reference", "cooking", "fiction".
4. similar to a web browser with bookmarks management. e.g. group colour, display tab order.
5. grouping to be displayed in Sumatra as tabs but within a named group, coloured group tab or theme.

**Comment by GitHubRulesOK** [comment]
Hmm there was an experimental grouping of documents (worked fairly well) but time for debugging one feature (caused several side issues) distracted from building up annotation so reluctantly dropped.

Main issue is every body and their siblings wants to be "different " so it becomes a Bodleian Library system with little time for fixing the viewing oddities. 

SumatraPDF can easily be used by Document Library systems as their viewer for multiple formats.
